docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -398,13 +398,49 @@ saved object ID of the action.
 
 |==============================================
 
+Optional fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|frequency |String a|Object containing action’s frequency:
+
+* `summary` (Boolean, required): Defines whether notification will be sent for summary of alerts or for each generated alert individually.
+
+* `notifyWhen` (String, required`): Defines how often alerts generate actions. Valid values are:
+
+** `onActiveAlert`: Actions run when the alert becomes active and at each check interval while the rule conditions are met.
+** `onThrottleInterval`: Actions run when the alert becomes active and at the interval specified in the throttle property while the rule conditions are met.
+
+* `throttle` (String, optional): Defines how often an alert generates repeated actions. This custom action interval must be specified in seconds, minutes, hours, or days. For example, `10m` or `1h`. This property is used only if `notifyWhen` is `onThrottleInterval`.
```

Comment:
```suggestion
* `throttle` (String, optional): Defines how often an alert prompts repeated actions. This custom action interval can be specified in seconds, minutes, hours, or days. For example, `10m` or `1h`. This property is used only if `notifyWhen` is `onThrottleInterval`.
```

Reply:
Here as well, we might need to use `a rule prompts` instead of `an alert prompts`

Reply:
Thanks, I'll make this change! I'm also going to make one small edit to the verb because I think "runs" is more appropriate here if we're talking about a rule triggering repeated actions. 

```suggestion
* `throttle` (String, optional): Defines how often rule run repeated actions. This custom action interval must be specified in seconds, minutes, hours, or days. For example, `10m` or `1h`. This property is used only if `notifyWhen` is `onThrottleInterval`.
```

Reply:
This looks great to me! Thank you!